src/lara.h

```diff
@@ -3105,9 +3105,15 @@ struct Lara : Character {
             || state == STATE_MIDAS_USE
             || state == STATE_MIDAS_DEATH
             // make me sick!
-            // || state == STATE_BACK_JUMP
-            // || state == STATE_LEFT_JUMP
-            // || state == STATE_RIGHT_JUMP
+            || state == STATE_BACK_JUMP
```

Comment:
the same, we should detach head animation in these cases because players will feel sick for virtual flips %)